.gitattributes

```diff
@@ -0,0 +1,2 @@
+/mvnw text eol=lf
+*.cmd text eol=crlf
```

Comment:
Readme file also should be added that will be used by Ops with information how to use this tool